tests/unit/test_file_handle_cache.py

```diff
@@ -37,26 +33,25 @@ def genFileName(x):
         self.setMaxCacheSize(9)
 
         # Build a list of 10 files and add their handles to the cache
-        fileList = map(genFileName, range(0, 10))
+        fileList = list(map(genFileName, list(range(0, 10))))
```

Comment:
Both "list"s are unnecessary

Reply:
The outer list is necessary for the later indexing, the second inner list can indeed be removed.